src/nb_clean/__init__.py

```diff
@@ -227,7 +234,11 @@ def check_notebook(
                 print(f"{prefix}: outputs")
                 is_clean = False
 
-    if not preserve_notebook_metadata:
```

Comment:
Following up on my comment above about having both `remove_notebook_metadata` and `preserve_notebook_metadata`, by removing this `--preserve-notebook-metadata` is no longer observed which breaks existing uses.